client/src/rpb/granular_view.js

```diff
@@ -100,70 +111,70 @@ class GranularView extends React.Component {
             nick,
             {
               index: idx + 2,
-              header: fully_qualified_name,
+              header: table.get_col_header(
+                {
+                  nick,
+                  fully_qualified_name,
+                },
+                dimension
+              ),
               is_searchable:
                 is_searchable && !is_matched_undefined(non_dept_key_cols, nick),
               is_summable:
                 is_summable && !is_matched_undefined(data_columns, nick),
-              formatter: type,
+              formatter: nick === "dept" ? "wide-str" : type,
```

Comment:
Is this necessary? Don't all the dept columns have type "wide-str" already?

Reply:
oh right I think I did that before I made `dept_and_legal_cols`